server/src/mindustry/server/ServerControl.java

```diff
@@ -335,14 +345,17 @@ protected void registerCommands(){
             }
         });
 
-        handler.register("maps", "Display all available maps.", arg -> {
+        handler.register("maps", "[all] [full]", "Display available maps. 'all' to include default maps. 'full' to show full map names.", arg -> {
+            boolean all = Arrays.asList(arg).contains("all");
```

Comment:
This is a bizarre way to handle arguments, `all` should be a boolean argument with false as a default, true if there's an argument and `arg[0].equals("true")`